controllers/cloud.redhat.com/version.txt

```diff
@@ -1 +1 @@
-master
\ No newline at end of file
+v0.29.0-15-gdfa3d6e
```

Comment:
This is another auto-generated change you want to revert: `git checkout origin/master controllers/cloud.redhat.com/version.txt`